troubleshooting.md

```diff
@@ -726,3 +726,21 @@ And then re-add the connection (removing the old one if necessary):
 And now this should work:
 
 `podman-remote info`
+
+---
+### 28) CNI networking fails in Podman v2.2.1 and earlier.
```

Comment:
This is only RHEL. Anything that is not RHEL will pull an image automatically to accomplish this. Also, version number is wrong - I believe 2.2.1 was the earliest that offered rootless CNI functionality, and 3.1 is the last that required the image, with 3.2 moving to the network namespace approach that does not require external dependencies (beyond CNI itself)